### ISSUE TITLE
Akka Typed Support

### ISSUE THREAD

**Issue by hudclark** [SwissBorg/lithium#98]
Using `"com.typesafe.akka" %% "akka-cluster-sharding-typed" % "2.6.4"`

When running lithium with the following config
```
akka {
  actor {
    provider = "cluster"
    no-serialization-verification-needed-class-prefix = [
      "akka.",
      "com.swissborg.lithium"
    ]
  }
  cluster.downing-provider-class = "com.swissborg.lithium.DowningProviderImpl"
}
com.swissborg.lithium {
  active-strategy = "keep-oldest"
  stable-after = 30s
  keep-oldest {
    down-if-alone = yes
  }
}

```

I see the following log on startup
```
2020-04-22 09:44:01,562 INFO  com.swissborg.lithium.DowningProviderImpl - Starting Lithium with the keep-oldest strategy.
2020-04-22 09:44:01,664 INFO  akka.actor.LocalActorRef - Message [akka.cluster.ClusterEvent$CurrentClusterState] from Actor[akka://MySystem/system/cluster/core/publisher#-317651006] to Actor[akka://MySystem/system/cluster/core/daemon/downingProvider/split-brain-reporter/reachability-reporter#-978528538] was unhandled. [1] dead letters encountered. This logging can be turned off or adjusted with configuration settings 'akka.log-dead-letters' and 'akka.log-dead-letters-during-shutdown'.
```

I do not see the `unhandled` warning when running lithium on akka-classic.

Is this message cause for concern or can it be ignored? @DennisVDB

**Comment by DennisVDB** [comment]
It seems I introduced the bug in #88. It shouldn't be harmful.

I left a `cluster.sendCurrentClusterState(self)` around that's not used. However, I'm a bit puzzled about why it doesn't happen on akka-classic. 

Both with classic  and typed you are using the same version of Lithium?

**Comment by hudclark** [comment]
Thanks for the speedy response!

 > Both with classic and typed you are using the same version of Lithium?

Sorry, no. The classic application uses lithium `0.10.0` and the typed application is on `0.11.1`

**Comment by DennisVDB** [comment]
Ah that's it then. #88 is not part of `0.11.0`. 
I'll probably push a fix by tonight.

**Comment by DennisVDB** [comment]
I've released v0.11.2 with the fix.
